go.mod

```diff
@@ -252,29 +258,60 @@ replace (
 	github.com/go-openapi/loads => github.com/go-openapi/loads v0.19.4
 	github.com/go-openapi/runtime => github.com/go-openapi/runtime v0.19.4
-	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
-	github.com/go-openapi/strfmt => github.com/go-openapi/strfmt v0.19.3
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.5
+	github.com/go-openapi/strfmt => github.com/go-openapi/strfmt v0.19.5
 	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
-	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.5
+	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.8
 	github.com/go-ozzo/ozzo-validation => github.com/go-ozzo/ozzo-validation v3.5.0+incompatible
 	github.com/go-stack/stack => github.com/go-stack/stack v1.8.0
+	github.com/go-toolsmith/astcast => github.com/go-toolsmith/astcast v1.0.0
+	github.com/go-toolsmith/astcopy => github.com/go-toolsmith/astcopy v1.0.0
+	github.com/go-toolsmith/astequal => github.com/go-toolsmith/astequal v1.0.0
+	github.com/go-toolsmith/astfmt => github.com/go-toolsmith/astfmt v1.0.0
+	github.com/go-toolsmith/astinfo => github.com/go-toolsmith/astinfo v0.0.0-20180906194353-9809ff7efb21
+	github.com/go-toolsmith/astp => github.com/go-toolsmith/astp v1.0.0
+	github.com/go-toolsmith/pkgload => github.com/go-toolsmith/pkgload v1.0.0
+	github.com/go-toolsmith/strparse => github.com/go-toolsmith/strparse v1.0.0
+	github.com/go-toolsmith/typep => github.com/go-toolsmith/typep v1.0.0
+	github.com/gobuffalo/here => github.com/gobuffalo/here v0.6.0
+	github.com/gobwas/glob => github.com/gobwas/glob v0.2.3
 	github.com/godbus/dbus/v5 => github.com/godbus/dbus/v5 v5.0.3
+	github.com/gofrs/flock => github.com/gofrs/flock v0.0.0-20190320160742-5135e617513b
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 	github.com/golang/freetype => github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0
 	github.com/golang/glog => github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/groupcache => github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
 	github.com/golang/mock => github.com/golang/mock v1.4.4
 	github.com/golang/protobuf => github.com/golang/protobuf v1.4.3
+	github.com/golangci/check => github.com/golangci/check v0.0.0-20180506172741-cfe4005ccda2
+	github.com/golangci/dupl => github.com/golangci/dupl v0.0.0-20180902072040-3e9179ac440a
+	github.com/golangci/errcheck => github.com/golangci/errcheck v0.0.0-20181223084120-ef45e06d44b6
+	github.com/golangci/go-misc => github.com/golangci/go-misc v0.0.0-20180628070357-927a3d87b613
+	github.com/golangci/goconst => github.com/golangci/goconst v0.0.0-20180610141641-041c5f2b40f3
+	github.com/golangci/gocyclo => github.com/golangci/gocyclo v0.0.0-20180528134321-2becd97e67ee
+	github.com/golangci/gofmt => github.com/golangci/gofmt v0.0.0-20190930125516-244bba706f1a
+	github.com/golangci/golangci-lint => github.com/golangci/golangci-lint v1.21.0
```

Comment:
I think this is bringing in the most transitive deps, is only depended on by sigs.k8s.io/kustomize/api, and shouldn't be a runtime dependency. could https://github.com/kubernetes-sigs/kustomize/tree/master/api/internal/tools be isolated like https://github.com/kubernetes-sigs/kustomize/tree/master/hack was?

Reply:
Done.